_quarto.yml

```diff
@@ -39,10 +39,12 @@ website:
         href: cheatsheet.qmd      
       - section: how-tos/index.qmd
         contents: 
-      #    - section: how-tos/find-data/index.md
-      #      contents:
-          - text: "Find Data"
-            href: how-tos/find-data/earthdata_search.md
+          - section: how-tos/find-data/index.md
+            contents:
+              - text: "Earthdata Search"
+                href: how-tos/find-data/earthdata_search.md
+              - text: "Programmatic Search" 
+                href: how-tos/find-data/programmatic.md                
```

Comment:
Turn this into .ipynb.

Reply:
Hi @asteiker this is taken care of now, see our most recent commit in this PR